### ISSUE TITLE
WCAG 2 level AA recommendation in the message

### ISSUE THREAD

**Issue by ankitwww** [HridoyHazard/BetterColor#70]
**Describe the Feature you want to end**
Currently, the user has to depend on her/his judgment for deciding the right "Color Readability". 
This can be tricky/wrong in some situations as this is a manual and partially subjective decision. 
Rather than just leaving the user on her taste/judgment - we should also give a hint of whether the current colors pass the WCAG 2 level AA contrast ratio thresholds.

**Expected behavior**
Users should be informed about the current contrast ratio.

**Screenshots**
A possible solution is to include the hint in the current message text.
![image](https://user-images.githubusercontent.com/14895768/194478984-30c2df30-e20d-4106-aabf-7e4021bd00ae.png)

**Additional context (optional )**
- Read more about [WCAG 2 contrast threshold](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**Comment by ankitwww** [comment]
hey @HridoyHazard - do you think this feature will be useful for the users? Happy to implement this.
